samples/system-test/samples.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const {assert} = require('chai');
+const cp = require('child_process');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const cwd = path.join(__dirname, '..');
+
+describe('Sample Integration Tests', () => {
+  it('should run quickstart.js', async () => {
+    try {
+      execSync('node ./quickstart.js billingAccounts/000000-000000-000000', {
+        cwd,
+      });
+    } catch (err) {
+      assert.include(err.stderr, 'PERMISSION_DENIED');
```

Comment:
Is the API under whitelist right now?

Reply:
It is, but also our testing accounts are tied to a universal billing account that they themselves don't have access to. I think writing integration tests for this API is going to be extremely difficult.